.github/workflows/terraform.yml

```
@@ -136,4 +136,9 @@ jobs:
       - name: terraform-apply
         id: apply
         if: inputs.apply
-        run: '[[ "$DESTROY" == "true" ]] && terraform destroy --auto-approve || terraform apply --auto-approve' # TODO: Adjust apply to pull the plan.out from artifact
\ No newline at end of file
+        run: |
+          if [[ "$DESTROY" == "true" ]]; then \
```

https://yaml-multiline.info/
```suggestion
          if [[ "$DESTROY" == "true" ]]; then
```